### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by Andresgue0913**

### CODE REVIEW

File: components/Form.jsx [hunk 1]
```diff
@@ -81,13 +83,14 @@ const Form = ({formData, forNewMovie = true}) => {
...
-        router.push("/");
+        return data.movie;
+        // router.push("/");
       }
     } catch (error) {
       console.log(error);
     }
   };
-
+console.log(selectedOption)
```
Comment (amingomezd) [inline components/Form.jsx]: remove the console.log

File: components/Form.jsx [hunk 2]
```diff
@@ -52,16 +53,17 @@ const Form = ({formData, forNewMovie = true}) => {
...
+        return data.data;
+        //   router.push("/");
       }
     } catch (error) {
-      console.log(error)
-  }
-  }
+      console.log(error);
+    }
+  };
```
Comment (amingomezd) [inline components/Form.jsx]: You can extract putData into a js module in "lib/movieApi.js" file and call it as updateMovie()

File: components/Form.jsx [hunk 3]
```diff
@@ -81,13 +83,14 @@ const Form = ({formData, forNewMovie = true}) => {
...
         }
       } else {
-        router.push("/");
+        return data.movie;
+        // router.push("/");
       }
     } catch (error) {
       console.log(error);
     }
   };
```
Comment (amingomezd) [inline components/Form.jsx]: You can extract postData into a js module in "lib/movieApi.js" file and call it as saveMovie()

File: components/Form.jsx (hunk 1 above)
Comment (Andresgue0913) [reply components/Form.jsx]: done

File: components/Form.jsx (hunk 2 above)
Comment (Andresgue0913) [reply components/Form.jsx]: done

File: components/Form.jsx (hunk 3 above)
Comment (Andresgue0913) [reply components/Form.jsx]: done

**Pull request merged**